modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/google-maps/google-maps.tsx

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {Loader} from '@googlemaps/js-api-loader';
+
+/**
+ * @description Load google global variable asynchronously
+ * @returns {void} Google Maps Autocomplete Instance
+ */
+
+type _adaptGoogleMapsAddressIntoAddress = (
+	addressComponents: any
+) => {
+	city: string;
+	country: string;
+	state: string;
+	street: string;
+	streetNumber: string;
+	zip: string;
+};
+const setup = (GOOGLE_API: any) => {
+	try {
+		const googleMapsLoader = new Loader({
+			apiKey: GOOGLE_API,
+			libraries: ['places'],
+		});
+
+		googleMapsLoader.load();
+	}
+	catch (error) {
+		console.warn(error);
+	}
+};
+
+/**
+ * @param {HTMLElement} input - Pass HTML element
+ * @returns {any} Google Maps Autocomplete Instance
+ */
+const autocomplete = (input: any) => {
+	const google = window.google;
+
+	if (!google) {
+		throw new Error(
+			'google is not defined. Please check the Google Maps API key within System Settings and ensure a valid API key is entered'
+		);
+	}
+
+	if (!input) {
+		throw new Error(
+			'No HTMLElement was found as input. Ensure a valid HTMLElement reference is passed!'
+		);
+	}
+
+	// Prevent crashes if the user hits enter in a autocomplete search
+
```

Comment:
Please, remove this white line
```suggestion
```

Reply:
We can't remove it, it's added by the save configurations.